src/theme.ts

```diff
@@ -16,6 +16,7 @@ const defaultTheme: ThemeInterface = {
     medium: '1024px',
     large: '1200px',
   },
+  background: 'var(--table)',
```

Comment:
If this is going to be the theme.background - we should name the css var something more like --background, or something similar.

Reply:
that makes more sense, updated!